website/content/api-docs/secret/pki.mdx

```diff
@@ -1678,7 +1680,7 @@ $ curl \
 }
 ```
 
-### OCSP request
+### OCSP request  <EnterpriseAlert product="vault" inline />
```

Comment:
OCSP is in OSS. Only the unified OCSP endpoint is enterprise-only.

Reply:
i.e., the ones with `local` below are in both OSS+ENT, the paths with `Unified` below are in ENT only.

Reply:
Okay. I removed the tag since there's a separate note about unified OCSP a little farther down in the doc.

Reply:
Thank you!